qiita_pet/templates/upload.html

```diff
@@ -96,7 +96,7 @@
 
 {% block content %}
 
-{% if (level != 'info' and 'Retrieving remote files:' not in message) or not message %}
+{% if (level != 'info' or 'Retrieving remote files:' not in message) or not message %}
```

Comment:
Does "level != 'info' or 'Retrieving remote files:' not in message" still need to be in parentheses, if they are now all OR conditions?